invenio_communities/assets/semantic-ui/js/invenio_communities/details_search/components.js

```diff
@@ -124,11 +134,44 @@ export const CommunityRecordResultsGridItem = ({ result }) => {
   );
 };
 
-export const CommunityRecordSearchBarContainer = () => {
+export const CommunityRecordSearchAppLayout = (props) => {
```

Comment:
```suggestion
export const CommunityRecordSearchAppLayout = ({config, ...}) => {
```
to keep the interface clean